notebook/gen_ai_jss.ipynb

```diff
@@ -0,0 +1,818 @@
+    "\n",
+    "This notebook assumes that you have already deployed this solution using either the [Terraform script]() **TODO: fix target for link** or using the [Solutions console](https://console.cloud.google.com/products/solutions/catalog). During this deployment, several actions required to run this solution were performed on your behalf:\n",
+    "\n",
+    "1. The [Cloud Function](https://console.cloud.google.com/functions/list) was deployed.\n",
+    "\n",
+    "2. The [EventArc trigger](https://console.cloud.google.com/eventarc/triggers) was applied to the input Cloud Storage bucket.\n",
+    "\n",
+    "3. The following APIs were enabled for you: \n",
+    "\n",
+    "   - [Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com)\n",
+    "   - [BigQuery API](https://console.cloud.google.com/flows/enableapi?apiid=bigquery.googleapis.com)\n",
+    "   - [Cloud Vision API](https://console.cloud.google.com/flows/enableapi?apiid=vision.googleapis.com)\n",
+    "\n",
+    "\n",
+    "<div style=\"background-color:rgb(150,200,255); padding:2px;\"><strong>Note:</strong> It is recommended to run this notebook from <a href=\"https://console.cloud.google.com/vertex-ai/workbench/\">Vertex AI Workbench</a>. If you are running this notebook locally instead, you need to install the <a href=\"https://cloud.google.com/sdk\" target=\"_blank\">Cloud SDK</a>.</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WReHDGG5g0XY"
+   },
+   "source": [
+    "#### Set your project ID\n",
+    "\n",
+    "**If you don't know your project ID**, try the following:\n",
+    "* Run `gcloud config list`.\n",
+    "* Run `gcloud projects list`.\n",
+    "* See the support page: [Locate the project ID](https://support.google.com/googleapi/answer/7014113)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "oM1iC_MfAts1"
+   },
+   "outputs": [],
+   "source": [
+    "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+    "\n",
+    "# Set the project id\n",
+    "! gcloud config set project {PROJECT_ID}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "region"
+   },
+   "source": [
+    "#### Region\n",
+    "\n",
+    "You can also change the `REGION` variable used by Vertex AI. Learn more about [Vertex AI regions](https://cloud.google.com/vertex-ai/docs/general/locations)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "region"
+   },
+   "outputs": [],
+   "source": [
+    "REGION = \"us-central1\"  # @param {type: \"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sBCra4QMA2wR"
+   },
+   "source": [
+    "### Authenticate your Google Cloud account\n",
+    "\n",
+    "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below."
```

Comment:
Maybe users wonder where they can run this notebook. Cloud shell could be an easy option. How about list out the few choices to run this colab?

Reply:
Good point!

They should ideally run this in Workbench. However, it is *possible* that they would try to run it directly from GitHub. We should probably think through where / how users access this notebook.

Reply:
Revised comment.